netbox/dcim/forms/connections.py

```diff
@@ -103,4 +103,10 @@ def clean(self):
             self.instance.a_terminations = self.cleaned_data['a_terminations']
             self.instance.b_terminations = self.cleaned_data['b_terminations']
 
+            if a_type == b_type and self.instance.a_terminations and self.instance.b_terminations:
```

Comment:
I've not checked but is this an issue on the API as well?

Reply:
moved to cable clean